cp23/hpi-fun2/cubes/problem.tex

```diff
@@ -0,0 +1,32 @@
+\input{template.tex}
+
+\begin{document}
+
+\makeheader
+
+``Setz dich nicht so nah vor den Fernseher - du bekommst noch viereckige Augen!''
+``Don't sit so close to the TV - you'll get square eyes!''
+
+You were told often enough by your parents, but of course you didn't believe them.
+Now, staring on computer screen all day to solve compettive programming tasks, the threat has turned into bitter reality.
+
+You only see the world as 1cm x 1cm x 1cm large aligned blocks, even after switching profession to be a meterologist, your sight hasn't cleared.
+
+One day, your new boss asks you to calculate the surface area of a large cloud on the sky, so the National Weather Modification Service can shoot the right amount of Silver iodide into the sky to let the cloud rain down.
+
+Can you calculate the surface area of the water drops in the cloud, which you only see as little cubes?
```

Comment:
But isn't it more the surface area of the cloud (consisting of the little cubes)?

Also more important:
Is the cloud connected?